packages/create-sitecore-jss/src/templates/angular-xmcloud/src/app/components/column-splitter/column-splitter.component.ts

```diff
@@ -0,0 +1,47 @@
+﻿import { Component, OnInit } from '@angular/core';
+import { SxaComponent } from '../sxa.component';
+
+@Component({
+  selector: 'app-column-splitter',
+  templateUrl: './column-splitter.component.html',
+})
+export class ColumnSplitterComponent extends SxaComponent implements OnInit {
+  ngOnInit() {
```

Comment:
ngOnInit is not needed if you are not defining something there, the same for "OnInit" interface